test/unit/Makefile

```diff
@@ -4,7 +4,7 @@ OBJDIR ?= objs/$(ARCH)
 .PHONY: all clean
 
 all: Makefile.include
-	git submodule update --init --rebase
+	(cd ../.. && git submodule update --init --rebase)
 	$(MAKE) -C $(OBJDIR)
 
```

Comment:
`git rev-parse --show-toplevel` will also give you the top level directory if you want to get fancy.  Also is the extra '(' shell necessary?

Reply:
Good points